sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BlobSasImplUtil.java

```diff
@@ -110,7 +110,7 @@ public BlobSasImplUtil(BlobServiceSasSignatureValues sasValues, String container
             throw logger.logExceptionAsError(
                 new IllegalArgumentException("'snapshot' and 'versionId' cannot be used at the same time."));
         }
-        this.version = sasValues.getVersion();
+        this.version = null;
```

Comment:
Can you comment these lines to explain we set it to null to force the latest version?